scripts/run_test.sh

```diff
@@ -68,116 +68,104 @@ function create_kubeplus(){
-            kubectl logs $pod
-            exit 1
-        fi
-        result=$(kubectl -n default get pods -lapp=postgres1  -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
-        echo $result
-        sleep $sleep_time
-        echo "Creating postgres ..."
-        kubectl get pods -n default
-        count=$((count + 1))
-    done
-    echo "Successfully created postgres instance!"
-    echo ""
+function test_explain() {
+    echo "Testing explain endpoint ..."
+    kubectl get --raw "/apis/platform-as-code/v1/explain?kind=Moodle"  | python -m json.tool
+    if [ $? -eq 0 ]; then
+        echo "Successfully called explain endpoint!"
+        echo ""
+    else
+        echo "Unable to call explain endpoint!"
+        echo ""
+        exit 1
+    fi
+    kubectl get --raw "/apis/platform-as-code/v1/explain?kind=Moodle.MoodleSpec"  | python -m json.tool
+    if [ $? -eq 0 ]; then
+        echo "Successfully called explain endpoint!"
+        echo ""
+    else
+        echo "Unable to call explain endpoint!\n"
+        echo ""
+        exit 1
+    fi
+
 }
-function delete_postgres() {
-    echo "Deleting postgres instance ..."
-    kubectl delete -f $PROJECT_HOME/examples/postgres/postgres1.yaml
-    timeout=240
-    sleep_time=3
-    loops=$((timeout/sleep_time))
-    count=0
-    result=$(kubectl -n default get pods -lapp=postgres1  -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
-    pod=`kubectl get pods | grep -v postgres-operator | grep postgres | awk '{print $1}'`
-    while [[ "$result" ]]; do
-        if [[ "$count" -gt $loops ]]; then
-            echo "Unable to delete postgres."
-            echo ""
-            echo "Operator logs: "
-            kubectl logs $pod
-            exit 1
-        fi
-        result=$(kubectl -n default get pods -lapp=postgres1  -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
-        sleep $sleep_time
-        echo "Deleting postgres..."
-        kubectl get pods -n default
-        count=$((count + 1))
-    done
-    echo "Successfully deleted postgres instance!"
-    echo ""
+function test_man() {
+    # echo "Testing man endpoint ..."
+    desired="NAME ===== Moodle Implementation Constants ========================= Moodle Operator uses following values in its code. These cannot be changed. - Moodle.Deployment.Pod.VolumeMounts.MountPath=/opt/moodledata - Moodle.PersistentVolume.StorageClass=manual - Moodle.PersistentVolume.Capacity=1Gi Usage Guidelines ================= In order to use Moodle Custom Resource you need to first create a MySQL Instance. You can either use MySQL Operator to create an instance or use a regular MySQL Deployment. Moodle Operator supports namespaces. When creating a Moodle Custom Resource you can specify the namespace in metadata.namespace attribute. If namespace attribute is not specified, default namespace is assumed. Check the implementation details section to know about some of the implementation choices and decisions that have been made by the Moodle Operator."
```

Comment:
Don't need to worry about the actual text. It may change. Just assert the response status and code (200 OK).

Reply:
Ok good idea, that's what I was thinking too the text may change